utils/utils.ts

```diff
@@ -300,10 +300,12 @@ export async function waitNewStakingRound(maxBlocks: number = 0) {
   let currentBlockNumber: number;
   const api = getApi();
   const parachainStakingRoundInfo = await api?.query.parachainStaking.round();
-  const sessionLength = parachainStakingRoundInfo.length.toNumber();
+  const sessionLength = parachainStakingRoundInfo.encodedLength;
```

Comment:
plz check this value, because lenght property do must exist.
![image](https://github.com/mangata-finance/mangata-e2e/assets/15139384/3ede62dd-982b-4b81-ac47-f2884f2d6dd8)

Reply:
I don't remember why I changed it, but the previous version works normally. Returned.